build.gradle

```diff
@@ -62,4 +64,10 @@ dependencies {
 	testCompile group: 'org.apache.camel', name: 'camel-test-spring', version: camelVersion
 	testCompile('org.springframework.boot:spring-boot-starter-test')
 	testCompile('org.assertj:assertj-core')
+	testCompile "com.github.tomakehurst:wiremock-standalone:2.18.0"
+
+	compile "com.google.code.gson:gson:2.8.5"
+	compile "org.json:json:20090211"
```

Comment:
What are you using this for ? (x2) no other services need to use it AFAIK.

Reply:
I removed all imports except the gson, which I use to validate the json blob.